src/qibolab/platform.py

```diff
@@ -352,6 +352,123 @@ def create_tii_rfsoc4x2(runcard, address=None):
     return platform
 
 
+def create_tii_zcu111(runcard, address=None):
+    """Create platform using QICK project on the ZCU111 board and EraSynth local oscillator for the Readout
+    IPs and other instrument related parameters are hardcoded in ``__init__`` and ``setup``.
+    Args:
+        runcard (str): Path to the runcard file.
+        address (str): Address and port for the QICK board.
+            If ``None`` it will attempt to connect to TII instruments.
+    """
+    # from qibolab.instruments.dummy_oscillator import (
+    #    DummyLocalOscillator as LocalOscillator,
+    # )
+    from qibolab.instruments.erasynth import ERA
+    from qibolab.instruments.rfsoc import RFSoC as TII_ZCU111
+
+    # Create channel objects
+    channels = ChannelMap()
+
+    # QUBIT 0 # D2
+    # readout
+    channels |= ChannelMap.from_names("L3-30_ro")  # dac6
+    # feedback
+    channels |= ChannelMap.from_names("L2-4-RO_0")  # adc0
+    # drive
+    channels |= ChannelMap.from_names("L4-29_qd")  # dac3
+    # Flux
+    channels |= ChannelMap.from_names("L1-22_fl")
+    # QUBIT 1 # D1
+    # readout
+    channels |= ChannelMap.from_names("L3-30_ro")  # dac6
+    # feedback
+    channels |= ChannelMap.from_names("L2-4-RO_1")  # adc1
+    # drive
+    channels |= ChannelMap.from_names("L4-30_qd")  # dac4
+    # Flux
+    channels |= ChannelMap.from_names("L1-23_fl")  # dac0
+    # QUBIT 2 # D5
+    # readout
+    channels |= ChannelMap.from_names("L3-30_ro")
+    # feedback
+    channels |= ChannelMap.from_names("L2-4-RO_2")
+    # drive
+    channels |= ChannelMap.from_names("L4-31_qd")
+    # Flux
+    channels |= ChannelMap.from_names("L1-24_fl")
+
+    # Map controllers to qubit channels (HARDCODED)
+    # Qubit 0
+    # readout
+    channels["L3-30_ro"].ports = [("dac6", 6)]
+    # feedback
+    channels["L2-4-RO_0"].ports = [("adc0", 0)]
+    # drive
+    channels["L4-29_qd"].ports = [("dac3", 3)]
+    # Flux
+    channels["L1-22_fl"].ports = [("dac0", 0)]
+    # Qubit 1
+    # Readout
+    channels["L3-30_ro"].ports = [("dac3", 6)]
```

Comment:
Are you sure this makes sense? You're overwriting what you inserted in https://github.com/qiboteam/qibolab/pull/373/files#diff-530316cac79b0748150794894b769b53dab9677ec7fe34fbeec09c2b39143ef6R403

Reply:
Maybe you might want to extend the list:
```suggestion
    channels["L3-30_ro"].ports += [("dac3", 6)]
```